invenio_app_ils/records/permissions.py

```diff
@@ -21,26 +26,28 @@ class RecordPermission(Permission):
     - Delete access to admin and specified users.
     """
 
-    create_actions = ['create']
-    read_actions = ['read']
-    update_actions = ['update']
-    delete_actions = ['delete']
+    _actions = {'POST': 'create',
```

Comment:
Is it needed to remap the request method to custom actions?
Would it make sense to write `collect_needs` like:
```
if request.method is 'POST':
  do_stuff()
...
```

Reply:
yep, I was thinking about it too, but I decided to go with mapping per actions because:

- PUT and PATCH would require repeating the same array with permissions (unless we want to make it more flexible)
- request method names are not very human readable if someone does not know what do they mean, and I remember someone mentioning the 2nd line support being able to read the record's properties (I might be wrong about this one), but you are right and it is too CERN specific, it should be more generic